lib/chef/application.rb

```diff
@@ -27,7 +27,7 @@
 require "tmpdir" unless defined?(Dir.mktmpdir)
 require "rbconfig"
 require_relative "application/exit_code"
-require_relative "dist"
+require "chef-utils"
```

Comment:
Let's require the specific part of chef-utils we need vs. the whole gem

Reply:
where I put a plain `require "chef-utils"` as opposed to `chef-utils/dist` it's because there was another chef-utils function being used, in this case `.windows?`. I could make 2, more specific require statements tho, would that be better?

Reply:
I just realized I may have reverted his change when I rebased this. Do you want me to split this up to requiring ``chef-utils/dist`` and ``chef-utils/dsl/windows`` like it was before?

Reply:
I think based on the conversation I had with @lamont-granquist this morning, this is probably better. to leave as it.